.travis.yml

```diff
@@ -1,15 +1,12 @@
 language: scala
 scala:
-- 2.11.2
-- 2.10.4
+  - 2.11.8
 jdk:
-- oraclejdk7
-- openjdk7
+  - oraclejdk8
+  - oraclejdk7
+  - openjdk7
 install:
-- ./.travis-cache.sh
+  - ./.travis-cache.sh
```

Comment:
Can we delete this and see what happens? We can do the sudo: false thing to trigger the docker builds which should use some sort of caching for SBT or something.